spec/machine/RISCV64/Platform.thy

```diff
@@ -109,10 +109,13 @@ definition kernelELFPAddrBase :: machine_word
   where
   "kernelELFPAddrBase = physBase + 0x4000000"
 
+definition pptrTop :: machine_word
+  where
+  "pptrTop \<equiv> - (1 << 31)"
```

Comment:
we should at some stage do a restyle of Platform.thy on all arches

- [ ] add issue

Reply:
Much agreed.